libs/src/services/ethContracts/serviceProviderFactoryClient.js

```diff
@@ -29,7 +29,7 @@ class ServiceProviderFactoryClient extends GovernedContractClient {
     const sanitizedEndpoint = endpoint.replace(/\/$/, '')
 
     if (!this.isDebug && !Utils.isFQDN(sanitizedEndpoint)) {
-      throw new Error('Not a fully qualified domain name!')
+      // throw new Error('Not a fully qualified domain name!')
```

Comment:
remove this

Reply:
to bypass adding a localhost for dashboard!